css/src/components/message.scss

```diff
@@ -58,4 +60,16 @@ $message-border: $border-width solid $border !default;
 			background-color: $message-sender-background-color;
 		}
 	}
+
+	&.message-size-sm {
```

Comment:
Let's follow the naming convention, it should be `message-sm` ;)

Reply:
Renamed.

Reply:
@JialiYang-96 please don't resolve conversations yourself, it should be done by the person who opened it.

Reply:
Sorry. This is my first time contributing to this repo. I'll take care in the future.

Reply:
No worries :)
Resolving